Makefile

```diff
@@ -2,15 +2,17 @@
 #
 
 # You can set these variables from the command line.
-PYTHON        = python3
-VENVDIR       = ./venv
-SPHINXBUILD   = $(VENVDIR)/bin/sphinx-build
-SPHINXOPTS    = --fail-on-warning --keep-going
-BUILDDIR      = _build
-BUILDER       = html
-JOBS          = auto
-PAPER         =
-SPHINXLINT    = $(VENVDIR)/bin/sphinx-lint
+PYTHON       = python3
+VENVDIR      = ./venv
+UV           = uv
+SPHINXBUILD  = $(VENVDIR)/bin/sphinx-build
+SPHINXOPTS   = --fail-on-warning --keep-going
+BUILDDIR     = _build
+BUILDER      = html
+JOBS         = auto
+PAPER        =
+SPHINXLINT   = $(VENVDIR)/bin/sphinx-lint
+REQUIREMENTS = requirements.txt
```

Comment:
The main changes here are adding the `UV` and `REQUIREMENTS` lines.

The others have a space removed, so it's the same as the CPython/Docs and peps `Makefiles` and easier to diff.